runtime/Java/src/org/antlr/v4/runtime/Parser.java

```diff
@@ -29,6 +29,13 @@
  */
 package org.antlr.v4.runtime;
 
+import java.util.ArrayList;
```

Comment:
:bulb: Your code formatting settings are moving things around. You can prevent these types of changes from appearing in a pull request by reviewing each file of a commit prior to committing it, and undoing any changes which aren't clearly part of the change you are working on.